api/__init__.py

```diff
@@ -357,8 +357,16 @@ def get_aww_link():
 	return aww_link
         '''
         #由於selenium實在是無法在apache開起來 只好再開一台虛擬機當作api
-        aww_link = requests.get("http://104.198.88.24/get_aww_link").text
+        #沒關係 有我(request)
+        url = 'https://awwapp.com/api/v2/admin/boards/create'
+        data={
+            "secret":"5de8ee4f-db69-4c61-8375-b36222ff4db8",
+            "domain":"https://awwapp.com//b/sezdetukm"
+        }
+        result = session_requests.post(url,data=data)
```

Comment:
response below these and get  "boardLink"
-> 
{
	"success": true,
	"board": {
		"__v": 0,
		"key": "8bfebc11-b33a-46fe-b5d5-a9436c0cb4e9",
		"ttl": 7200000,
		"host": "awwapp.com",
		"plan": "personal",
		"_id": "5ba38130e13c3267876585ab",
		"isFrozen": false,
		"connectAllAsGuests": false,
		"boardSnapshotLink": "stblxhamn",
		"name": "Unnamed Board",
		"hasAudio": false,
		"subscription": "personal",
		"isPermanent": true,
		"isSnapshot": true,
		"enableClassroom": false,
		"boardLink": "uhnj3m2mz",
		"modifiedAt": "2018-09-20T11:14:56.403Z",
		"createdAt": "2018-09-20T11:14:56.403Z"
	}
}